datafusion/physical-expr/src/expressions/binary.rs

```diff
@@ -5117,31 +5117,31 @@ mod tests {
         let left = Arc::new(Int32Array::from(vec![Some(12), None, Some(11)])) as ArrayRef;
         let right =
             Arc::new(Int32Array::from(vec![Some(1), Some(3), Some(7)])) as ArrayRef;
-        let mut result = bitwise_and(left.clone(), right.clone())?;
+        let mut result = bitwise_and_dyn(left.clone(), right.clone())?;
```

Comment:
renamed these kernels to match the naming scheme in arrow-rs (where `_dyn` is used to refer to kernels that take `ArrayRef`